src/components/StyleGuide/Typography.mdx

```diff
@@ -1395,9 +1751,32 @@ Recommendation: maximum of 40 words (approx. 200 characters)
 
 ## Figma Reference
 
-For additional spacing information, please refer to the Figma Main file.
+For additional spacing information, please refer to the Reservoir Visual Design
```

Comment:
Should we add that this is for NYPL employees? The figma links are not public.

Reply:
Most, if not all, of the component story pages have links to the VDL, so this is no different.  I think it's fine as is.